mapstore2_adapter/plugins/geonode.py

```diff
@@ -608,3 +556,62 @@ def viewer_json(self, viewer, request):
             config_bbox = [ms2_bbox[0], ms2_bbox[2], ms2_bbox[1], ms2_bbox[3]]
             viewer['map']['bbox'] = config_bbox
         return viewer
+
+    def get_layer_featureInfo(self, layer):
```

Comment:
@marthamareal please rename the method to get_layer_featureinfotemplate

Reply:
@giohappy  i have updated, this.